.gitignore

```diff
@@ -1,2 +1,13 @@
 .idea
```

Comment:
Do we have some test coverage for windows?

I think we only runs the [github workflows on ubuntu](https://github.com/DataDog/datadog-traceroute/blob/main/.github/workflows/build.yml#L19), but would help to also run on windows WDYT?